dft/src/traits.rs

```diff
@@ -25,8 +25,19 @@ use crate::util::{coset_shift_cols, divide_by_height};
 /// parallel feature) different implementation may be faster. Hence depending on your use case
 /// you may want to be using `Radix2Dit, Radix2DitParallel, RecursiveDft` or `Radix2Bowers`.
 pub trait TwoAdicSubgroupDft<F: TwoAdicField>: Clone + Default {
-    // Effectively this is either RowMajorMatrix or BitReversedMatrixView<RowMajorMatrix>.
-    // Always owned.
+    /// The matrix type used to store the result of a batched DFT operation.
+    ///
+    /// This type represents a columnar matrix of field elements, typically used
+    /// to hold the evaluation results of multiple polynomials over a two-adic subgroup
+    /// or its coset. It is always owned and supports efficient access and transformation
+    /// patterns used in FFT-based algorithms.
```

Comment:
I don't think this matrix is columnar. (That seems to be a matrix with only a single column)
```suggestion
    /// This type represents a matrix of field elements, used to hold the evaluations 
    /// of multiple polynomials over a two-adic subgroup or its coset.
    /// It is always owned and supports efficient access and transformation
    /// patterns used in FFT-based algorithms.
```